packages/core/src/lib/markdown-it/patches/custom-component/htmlBlockRule.js

```diff
@@ -0,0 +1,146 @@
+const blockNames = require('markdown-it/lib/common/html_blocks');
+const { HTML_OPEN_CLOSE_TAG_RE } = require('./htmlRe');
+const inlineTags = require('./inlineTags');
+const vueReservedTags = require('./vueReservedTags');
+
+function initCustomComponentHtmlBlockRule(tagsToIgnore) {
+  // MODIFIED (MarkBind): Added 'special tags' and 'minimal panel' related rules
+
+  /* 
+   * Set up special tags that we need to ignore for markdown parsing.
+   * Custom patch for the api to escape content in certain special tags.
+   */
+  const specialTagsRegex = Array.from(tagsToIgnore)
+  .concat(['include|script|pre|style|variable|site-nav|markdown|md'])
+  .join('|');
+
+  // attr_name, unquoted ... attribute patterns adapted from markdown-it/lib/common/html_re
+  // Construct self-closing negative lookahead for special tags,
+  // because if matched, the endingSpecialTagRegex will roll down to find a non-existent closing tag
+  const attr_name = '[a-zA-Z_:][a-zA-Z0-9:._-]*';
+  const unquoted      = '[^"\'=<>`\\x00-\\x20]+';
+  const single_quoted = "'[^']*'";
+  const double_quoted = '"[^"]*"';
+  const attr_value  = '(?:' + unquoted + '|' + single_quoted + '|' + double_quoted + ')';
+  const attribute   = '(?:\\s+' + attr_name + '(?:\\s*=\\s*' + attr_value + ')?)';
+  const selfClosingNegativeLookahead = `(?!${attribute}*\\s*\/>)`;
+
+  const startingSpecialTagRegex = new RegExp(
+  `^<(${specialTagsRegex})${selfClosingNegativeLookahead}(?=(\\s|>|$))`, 'i');
+  const endingSpecialTagRegex = new RegExp(`<\\/(${specialTagsRegex})>`, 'i');
+
+  /*
+  * In MarkBind, we consider minimized panel components to be inline elements.
+  * Thus, we have to ensure that markdown-it parses minimized panel as inline element
+  * by using the regex below. 
+  */
+  const minimizedPanelStringRegex = '(panel.*minimized.*)';
+
+  // Forked and modified from 'markdown-it/lib/rules_block/html_block.js'
+
+  // An array of opening and corresponding closing sequences for html tags,
+  // last argument defines whether it can terminate a paragraph or not
+  const HTML_SEQUENCES = [
+    // MODIFIED (MarkBind): ignore special tags
+    [ startingSpecialTagRegex, endingSpecialTagRegex, true ],
+    [/^<(script|pre|style)(?=(\s|>|$))/i, /<\/(script|pre|style)>/i, true],
```

Comment:
My bad! 😓  I thought u were referring to merging with `escape-special-tags` in general. I have fixed it now! Instead of removing that line you highlighted above (which is adapted from `VuePress`), I opted to remove the `script`, `pre`, and `style` from the special tags regex instead as it seemed more appropriate. Let me know if I should do the opposite :-)